src/zeep/tornado/transport.py

```diff
@@ -113,8 +114,8 @@ def fetch(self, address, method, headers, message=None):
             'auth_username': auth_username,
             'auth_password': auth_password,
             'auth_mode': auth_mode,
-            'validate_cert': self.session.verify is not None,
-            'ca_certs': self.session.verify,
+            'validate_cert': bool(self.session.verify),
```

Comment:
The problem with the removed line is:
```python
False is not None
```
 will return `True`, which is not the wanted result.